src/Symfony/Bridge/Doctrine/ContainerAwareEventManager.php

```diff
@@ -149,17 +153,27 @@ public function removeEventListener($events, $listener): void
         }
     }
 
+    /**
+     * @deprecated since version 6.3
+     */
     public function addEventSubscriber(EventSubscriber $subscriber): void
     {
+        trigger_deprecation('symfony/doctrine-bridge', '6.3', 'Method "%s()" is deprecated.', __METHOD__);
```

Comment:
The notice should tell about the alternative (`use ... instead.`). Same for all deprecated methods

Reply:
ok !